CoinDock-app/src/Screens/Account/Account.js

```diff
@@ -8,37 +8,99 @@ import { useLogout } from "App/Api/auth";
 import "Shared/common-styles/common.css";
 
 function Account() {
-  const { data: account, isLoading } = useAccount();
-  const [logout] = useLogout();
-  const navigate = useNavigate();
+ const { data: account}= useAccount();
+ const accountDetails = account?.result?.user || {};
+ const [logout] = useLogout();
+ const navigate = useNavigate();
 
-  const handleLogoutClick = async () => {
-    try {
-      await logout().unwrap();
-      navigate("/login");
-    } catch (e) {
-      navigate("/login");
-    }
-  };
+ const accordianBasedAccountDetails = [
+   {
+     label:'Profile Settings',
+     fields:[
+       {
+       label:'Name',
+       fieldKey:'name',
+       type:'edit'
+      },
+       {
+        label:'DateofBirth',
+        fieldKey:'dateofbirth',
+        type:'edit'
+       },
+       {
+        label:'Country',
+        fieldKey:'country',
+        type:'edit'
+       },
+    ]
+   },
+   {
+     label:'Account Settings',
+     fields:[
+         {
+          label:'Email',
+          fieldKey:'email',
+          type:'edit'
+         },
+         {
+          label:'Change Password',
+          fieldKey:'changePassword',
+          type:'edit'
+         },
+         {
+          label:'Recovery Code',
+          fieldKey:'recoverycode',
+          navigate: "/recovery-codes-account",
+         },
 
-  return (
+    ]
+   },
+     {
+     label:'System Settings',
+     fields:[
+       {
+         label:'Primary currency',
+         fieldKey:'primarycurrency',
+       },
+       {
+        label:'Secondry currency',
+        fieldKey:'secondrycurrency',
+      }
+     ]
+     }
+  ]
+
+ const handleLogoutClick = async () => {
+  try {
+    await logout().unwrap();
+    navigate("/");
+  } catch (e) {
+    navigate("/")
```

Comment:
we should not navigate no other screen user logout fails. He has to stay in the same screen mostly.